CONTRIBUTING.md

```diff
@@ -194,11 +198,37 @@ Code Formatter] installed, you can apply formatting directly in IntelliJ.
 
 Note that only some sub-projects in the Elasticsearch project are currently
 fully-formatted. You can see a list of project that **are not**
-automatically formatted in [gradle/formatting.gradle](gradle/formatting.gradle).
+automatically formatted in
+[build-tools-internal/src/main/groovy/elasticsearch.formatting.gradle](build-tools-internal/src/main/groovy/elasticsearch.formatting.gradle).
+
+#### Imports
+
+It is also important that you configure the correct import ordering in
```

Comment:
Is there really no way to automate setting this up somehow? A lot of the other manual stuff is sort of "optional" but optimizing imports is something _everyone_ does and if someone misses this step they are going to be fighting with spotless _a  lot_.